src/renderer/containers/Onboard/Configure/ConfigureDockerPath.tsx

```diff
@@ -14,17 +16,26 @@ const Configure: React.FunctionComponent<IOwnProps & IInjectedProps> = (props) =
     const [valid, setValid] = useState(null);
 
     const savePath = async(): Promise<void> => {
-        if (await DockerPath.isValidPath(path)) {
+        const {filePaths} = await remote.dialog.showOpenDialog({
+            title:"Select Docker binary",
+            properties: ["openFile"]
+        });
+        setPath(filePaths[0]);
```

Comment:
if canceled this will throw

Reply:
Errors as invalid path. I guess that's ok?

Reply:
Maybe not

Reply:
I think you should just return from method if users cancels dialog instead of logging stacktraces